src/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -113,6 +114,12 @@ void CommandLine::DumpHelp(const char* program)
     printf("     Use disassembly tools from the CoreDisTools library\n");
     printf("\n");
 #endif // USE_COREDISTOOLS
+    printf(" -jitoption key=value\n");
+    printf("     Set the JIT option named \"key\" to \"value\" for JIT 1. NOTE: do not use a \"COMPlus_\" prefix!\n");
+    printf("\n");
+    printf(" -jit2option key=value\n");
+    printf("     Set the JIT option named \"key\" to \"value\" for JIT 2. NOTE: do not use a \"COMPlus_\" prefix!\n");
+    printf("\n");
```

Comment:
Do we really want the user model to allow specifying different flags to each JIT? Would you really want to pass something different to baseline and diff? Wouldn't it be inconvenient to have to duplicate what you are specifying twice?